front/public/locales/fr/errors.json

```
@@ -113,11 +113,13 @@
     },
     "macro_node": {
       "Database": "Erreur interne",
-      "NotFound": "Noeud {{node_id}} non trouvé."
+      "NotFound": "Noeud {{node_id}} non trouvé.",
+      "ScenarioNotFound": "Scenario '{{scenario_id}}' non trouvé"
```

Same for the other key trad
```suggestion
      "ScenarioNotFound": "Scénario '{{scenario_id}}' non trouvé"
```